src/content/en/fundamentals/performance/prpl-pattern/index.md

```diff
@@ -221,6 +222,12 @@ The combination of HTTP/2 and HTTP/2 server push provides the _benefits_ of
 bundling (reduced latency) without actual bundling. Keeping resources separate
 means they can be cached efficiently and be shared between pages.
 
+HTTP/2 Push needs to be utilized with care, as it forces data to the browser,
```

Comment:
My sense is that there is a place to warn people about the dangers of PUSH, but that the PRPL doc is not necessarily that place. I'd rather see that discussion happen in some doc specific to H/2 PUSH, with just a brief aside in this PRPL doc, along the lines of "H/2 PUSH works well in the context of PRPL, but you should exercise caution when adopting PUSH in other contexts -- see [other doc] for more."